tpm/tpm.go

```diff
@@ -76,9 +76,22 @@ func GetKeys(rw io.ReadWriter) ([]Handle, error) {
 	return unpackKeyHandleList(b)
 }
 
+// PcrExtend extends a value into the right PCR by index.
+func PcrExtend(rw io.ReadWriter, i uint32, p pcrValue) ([]byte, error) {
```

Comment:
I know Go really likes short names. However, for consistency with other argument names in this file, and because this function is exported, I'd like to see "i" and "p" expanded. Maybe pcrIndex and value?

I have the same comment about ReadPCR, below.